src/orion/core/cli/plot.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Module running the plot command
+========================
+
+Exposes the interface for plotting for command-line usage.
+
+"""
+import argparse
+import logging
+
+import orion.core.io.experiment_builder as experiment_builder
+from orion.core.cli import base as cli
+from orion.storage.base import get_storage
+import orion.plotting.base
+
+# c'est quoi que ça fait ça?
+#from orion.storage.base import get_storage
+
+
+log = logging.getLogger(__name__)
+DESCRIPTION = "Produce plots for Oríon experiments"
+
+
+
+def add_subparser(parser):
+    """Add the subparser that needs to be used for this command"""
+    plot_parser = parser.add_parser("plot", help=DESCRIPTION, description=DESCRIPTION)
+
+    cli.get_basic_args_group(plot_parser)
+
+    plot_parser.add_argument(
+        "kind",
+        type=str,
+        help="kind of plot to generate. "
+        " Pick one among ['lpi', 'partial_dependencies', 'parallel_coordinates', 'regret']"
+    )
+
+    plot_parser.add_argument(
+        "-t",
+        "--type",
+        type=str,
+        default="png",
+        help="type of plot to return. "
+        " Pick one among ['png', 'jpg', 'jpeg', 'webp', 'svg', 'pdf', 'html', 'json']"
+        " (default: png)",
+    )
+
+    plot_parser.add_argument(
+        "-o",
+        "--output",
+        type=str,
+        default="",
+        help="path where plot is saved. "
+        " Will override the `type` argument."
+        " (default is {exp.name}_{kind}.{type})"
+    )
+
+    plot_parser.add_argument(
+        "--scale",
+        type=float,
+        default=None,
+        help="more pixels, but same proportions of the plot. "
+        " Reference is 1.0."
```

Comment:
I guess greater than 1 means more high definition?

Reply:
Yes. I think it's natural to think that scale 2.0 means you double the pixels. I didn't think for a moment that it could be ambiguous, but I guess there's some ambiguity. Do you think it needs clarification?

Reply:
Maybe not. :)